core/server/src/main/java/alluxio/master/file/meta/LockedInodePath.java

```diff
@@ -71,6 +71,14 @@ public synchronized Inode getInode() throws FileDoesNotExistException {
   }
 
   /**
+   * @return the target inode which is guaranteed to exist
+   */
+  public synchronized Inode getInodeNoException() {
```

Comment:
I would prefer to catch the exception at the call site instead of introducing a new idiom into our code base